maps-compose/src/main/java/com/google/maps/android/compose/MapUpdater.kt

```diff
@@ -34,9 +34,11 @@ internal class MapPropertiesNode(
     var clickListeners: MapClickListeners,
     var density: Density,
     var layoutDirection: LayoutDirection,
+    contentPadding: PaddingValues
 ) : MapNode {
 
     init {
+        applyContentPadding(map, contentPadding)
         cameraPositionState.setMap(map)
```

Comment:
Could you add a brief comment here to inform a reader why padding must precede camera position?

```suggestion
        applyContentPadding(map, contentPadding)
        // set camera position after padding for correct centering
        cameraPositionState.setMap(map)
```